atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -36,8 +36,10 @@
 
     public static final String PRIMARY_KEY_CONSTRAINT_PREFIX = "pk_";
 
+    private static final int POSTGRES_NAME_LENGTH_LIMIT = 63;
     private static final int ORACLE_NAME_LENGTH_LIMIT = 30;
     public static final int ATLASDB_ORACLE_TABLE_NAME_LIMIT = AtlasDbConstants.ORACLE_NAME_LENGTH_LIMIT - PRIMARY_KEY_CONSTRAINT_PREFIX.length();
+    public static final int ATLASDB_POSTGRES_TABLE_NAME_LIMIT = AtlasDbConstants.POSTGRES_NAME_LENGTH_LIMIT - PRIMARY_KEY_CONSTRAINT_PREFIX.length();
```

Comment:
So Postgres does the truncation by default, but if we add a table mapper this means we store the truncated names (similar to Oracle) and throw when another table truncates to an existing name. However, the issue here is that Dbkvs users will have to migrate to get the changes or we will end up with another config flag for `useTableMapping`. I dont like these. Also, note that `PostgresTableNameMapper` will not be prioritized anytime soon. We expect users to have unique first 60 character table names.